Source/CMakeLists.txt

```diff
@@ -19,10 +19,11 @@ endif()
 add_definitions(-D__STDC_LIMIT_MACROS)
 add_definitions(-D__STDC_CONSTANT_MACROS)
 
-macro(add_dolphin_library lib srcs libs)
+function(add_dolphin_library lib srcs libs)
 	add_library(${lib} STATIC ${srcs})
 	target_link_libraries(${lib} PUBLIC ${libs})
-endmacro()
+	set_target_source_folder(${lib})
```

Comment:
No, it doesn't.
`function()` changes the context for variables, but it's still the same makefile. If it didn't work, then it wouldn't pass CI.